modules/ROOT/pages/6.5.1-release-notes.adoc

```diff
@@ -839,16 +843,16 @@ Previously, when Google Chrome was the host browser, there were circumstances wh
 
 When the insertion point was positioned after a table, it was placed in the `<body>` at offset 1 instead of within any of its children.
 
-As a result, when an element with a `contenteditable= "false"` attribute came after it, it was deemed invalid to insert a newline at that position. Consequently, no newline was added.
+As a result, when an element with a `contenteditable="false"` attribute came after it, it was deemed invalid to insert a newline at that position. Consequently, no newline was added.
 
-As of this release, {productname} now recognises this position as valid, and the expected root blocks are inserted.
+As of this release, {productname} now recognizes this position as valid, and the expected root blocks are inserted.
 
 === In some cases, exiting a `blockquote` element could fail when the insertion point was positioned at the end of the `blockquote` element.
 //#TINY-9794
 
 There was an issue with exiting a blockquote element when the insertion point was positioned at the end of the element. this issue arose because, when inserting a new block, the check to determine if the block is empty treated nodes containing a zero-width space (ZWSP) as non-empty.
```

Comment:
Typo -> ". this issue" to ". This issue"

Reply:
```suggestion
There was an issue with exiting a blockquote element when the insertion point was positioned at the end of the element. This issue arose because, when inserting a new block, the check to determine if the block is empty treated nodes containing a zero-width space (ZWSP) as non-empty.
```